docs/root/configuration/http/http_filters/grpc_json_reverse_transcoder_filter.rst

```diff
@@ -0,0 +1,55 @@
+.. _config_http_filters_grpc_json_reverse_transcoder:
+
+gRPC-JSON reverse transcoder
+============================
+
+* gRPC :ref:`architecture overview <arch_overview_grpc>`
+* This filter should be configured with the type URL ``type.googleapis.com/envoy.extensions.filters.http.grpc_json_reverse_transcoder.v3.GrpcJsonReverseTranscoder``.
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.grpc_json_reverse_transcoder.v3.GrpcJsonReverseTranscoder>`
+
+This is a filter which allows a gRPC client to send requests to Envoy and get proxied
+to a RESTful JSON API service. The HTTP mapping for the gRPC service has to be defined by
+`custom options <https://cloud.google.com/service-management/reference/rpc/google.api#http>`_.
+
+JSON mapping
+------------
+
+The protobuf to JSON mapping is defined `here <https://developers.google.com/protocol-buffers/docs/proto3#json>`_.
+
+.. _config_grpc_json_generate_proto_descriptor_set:
```

Comment:
this anchor is duplicating an existing one

Reply:
Fixed